packages/noports_core/lib/src/common/at_ssh_key_util/local_ssh_key_util.dart

```diff
@@ -47,8 +47,10 @@ class LocalSshKeyUtil implements AtSshKeyUtil {
       files[1].writeAsString(keyPair.publicKeyContents),
     ]).catchError((e) => throw e);
 
-    chmod(files[0].path, '600');
-    chmod(files[1].path, '644');
+    if (!Platform.isWindows) {
+      chmod(files[0].path, '600');
+      chmod(files[1].path, '644');
+    }
 
```

Comment:
These functions are posix only and don't work on windows, and I didn't have any file permission issues in Windows... It's possible the windows OpenSSH doesn't even check because of how painful it is to do in Windows